src/Media/Media_Subscriber.php

```diff
@@ -65,4 +67,19 @@ private function disable_responsive_images(): void {
 			$this->container->get( WP_Responsive_Image_Disabler::class )->disable_wordpress_filters();
 		}, 10, 0 );
 	}
+
+	private function oembed(): void {
+		if ( defined( 'TRIBE_DISABLE_YOUTUBE_NO_COOKIE' ) && TRIBE_DISABLE_YOUTUBE_NO_COOKIE === false ) {
```

Comment:
❓ 
Why not default it to not using cookies? Isn't that the way things are moving right now?

Reply:
... Nevermind. that is what you are doing. 🤦

Reply:
Oh yeah, this conditional is a bit confusing. I'll patch it up next week.